### PR TITLE
docs: add auth0 integration tutorial

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

**Comment by dlecan** [comment]
https://spacelephant.tpondemand.com/entity/2499

File: docs/integration/README.md [hunk 1]
```diff
@@ -94,3 +94,36 @@ In this example, <uniknameconnect/> is used to access Microsoft Office 365 accou
...
+### #8 auth0 Integration
+
+If you not have auth0 application yet, please follow the [Get Started](https://auth0.com/docs/quickstarts) auth0 tutorial to create one.
+
+Once your application is created, you can add an `Enterprise Connection`. Use `Open Id Connect` link.
+
+For <uniknameconnect/> configuration fill those required fields:
+- `Connection name`: It's an internal name (Ex: `unikname-connect`)
+- `Display name`: Unikname (Will display `Continue with Unikname`, on the connection button)
+- `Logo url`: you can add 20x20 <uniknameconnect/> logo image url
```
Comment (Nigui) [inline docs/integration/README.md]: we could add a link to one of our image (favicon ? )

File: docs/integration/README.md [hunk 2]
```diff
@@ -94,3 +94,36 @@ In this example, <uniknameconnect/> is used to access Microsoft Office 365 accou
...
 
+### #8 auth0 Integration
+
+If you not have auth0 application yet, please follow the [Get Started](https://auth0.com/docs/quickstarts) auth0 tutorial to create one.
+
+Once your application is created, you can add an `Enterprise Connection`. Use `Open Id Connect` link.
+
+For <uniknameconnect/> configuration fill those required fields:
+- `Connection name`: It's an internal name (Ex: `unikname-connect`)
+- `Display name`: Unikname (Will display `Continue with Unikname`, on the connection button)
```
Comment (Nigui) [inline docs/integration/README.md]: `@`Unikname ?

File: docs/integration/README.md [hunk 3]
```diff
@@ -94,3 +94,36 @@ In this example, <uniknameconnect/> is used to access Microsoft Office 365 accou
...
+- `Logo url`: you can add 20x20 <uniknameconnect/> logo image url
+- `Issuer URL`: `{unikname connect url}/oidc/.well-known/openid-configuration` (<uniknameconnect/> IDP url)
```
Comment (Nigui) [inline docs/integration/README.md]: we should duplicate url here.

File: docs/integration/README.md [hunk 4]
```diff
@@ -94,3 +94,36 @@ In this example, <uniknameconnect/> is used to access Microsoft Office 365 accou
...
+- `Client ID`: your organisation did in base64URL (Follow instructions bellow to generate your Client ID)
+- `Client Secret`: your client secret
+
+#### Generate you Client ID
+
+Encode your organization did in base64URL, you can easily find online encoders
```
Comment (Nigui) [inline docs/integration/README.md]: we could add tool link to help user. Like : http://www.base64url.com/

File: docs/integration/README.md [hunk 5]
```diff
@@ -94,3 +94,36 @@ In this example, <uniknameconnect/> is used to access Microsoft Office 365 accou
...
+
+Your did looks like: `did:unik:unid:{your UNIK token id}`
+
+Ex: `did:unik:unid:aad289fdfa8ab05fadf68f0f59b134f357e1d49a8bb5aeda3d2d45071777cd6f`
+
+Your Client ID should look like this: `ZGlkOnVuaWs6dW5pZDphYWQyODlmZGZhOGFiMDVmYWRmNjhmMGY1OWIxMzRmMzU3ZTFkNDlhOGJiNWFlZGEzZDJkNDUwNzE3NzdjZDZm`
+
+#### Activate your connection with <uniknameconnect/> team
+
+Contact <uniknameconnect/> team to activate your service.
```
Comment (Nigui) [inline docs/integration/README.md]: a direct email ?

**Review by Nigui — APPROVED** [review]
great ! It looks so simple ! 
I've made some enhancement comments :)

**Pull request merged**